scripts/render-metrics-gif.py

```
@@ -34,163 +46,467 @@
+        cell_w = 0.8
+        cell_h = 0.7
+        x_pad = 0.3   # left margin for labels
+        y_pad = 0.15   # bottom margin
+
+        # Compute total grid width needed
+        total_w = x_pad + max_in_phase * (cell_w + 0.15) + 3.0  # extra for right annotations
+
+        ax_kc.set_xlim(-0.2, total_w)
+        ax_kc.set_ylim(-0.5, n_active * (cell_h + 0.25) + 0.3)
+
+        for row_idx, (phase_global_idx, pname) in enumerate(reversed(active_phases)):
```

Ruff (0.15.12)

[warning] Loop control variable `phase_global_idx` not used within loop body

Rename unused `phase_global_idx` to `_phase_global_idx`

(B007)